python-sdk/src/astro/table.py

```diff
@@ -132,9 +132,10 @@ def row_count(self) -> Any:
         Return the row count of table.
         """
         db = create_database(self.conn_id)
-        tb = db.get_sqla_table(table=self)
-        query = select(func.count("*")).select_from(tb)
-        return db.run_sql(query).scalar()
+        result = db.run_sql(
+            f"select count(*) from {db.get_table_qualified_name(self)}"  # skipcq: BAN-B608
```

Comment:
@rajaths010494 can we not skip this?

Reply:
ok will fix this

Reply:
As discussed i am currently skipping this.